### DISCUSSION TITLE
cyberduck開啟mail問題

### DISCUSSION THREAD

**Discussion by fefefw** [bol-edu/HLS-SOC-Discussions#229] [General]
點擊mail後出現以下問題!
![螢幕擷取畫面 2024-09-06 183646](https://github.com/user-attachments/assets/66fa1eb8-0361-4bb1-ae5f-7c9fd00ac236)

**Comment by david870110** [comment]
可以不用使用Cyberduck 
直接整包拉下來就行了
如果想要使用，要去設定調整解除應用程式封鎖，google查詢一下詳細操作
- JIANG
